src/components/NetworkMapCanvas.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <div class="flex flex-col h-full">
+    <h3>Network Map</h3>
+    <div class="map-container">
+      <div class="wrapper">
+        <img ref="mapImage" src="/assets/mercator-map.png" alt="">
+        <canvas ref="mapCanvas"></canvas>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "NetworkMap",
+  data: function () {
+    return {
+      mapLngLeft: -180,
+      mapLngRight: 180,
+      mapLatBottom: -57,
```

Comment:
Do these need to be component data or can they be moved out to constants? Also, a comment to explain why `mapLatBottom` is -57 would help.

Reply:
will remove - this is from old map